azurelinuxagent/ga/exthandlers.py

```diff
@@ -1398,6 +1399,17 @@ def initialize(self):
         # Save HandlerEnvironment.json
         self.create_handler_env()
 
+    def should_create_default_placeholder(self, extension=None):
+        """
+        There's a bug in the AKS extension where they dont update the status file if it exists.
+        This violates the contract we have with extensions. Until they fix their extension,
+        we're going to skip creating a placeholder for them to ensure they dont have any downtime.
+        For all other extensions, we should create a
+        """
+
+        ignore_extension_regex = "Microsoft\\.AKS\\.Compute\\.AKS\\S*"
```

Comment:
minor: using a raw string may make the regex slighly more readable: r"Microsoft\.AKS\.Compute\.AKS\S*"

Reply:
Done